### PR TITLE
Common terms main page and interface overview from MichaelW branch

### PR DESCRIPTION
**Pull request by pedrosousabarreto**
This PR includes the commons terms page and the interfaces page overview extracted from MichaelW #39 branch;

Note, I've re-written the interface overview and added an ho to read this table section.

### CODE REVIEW

**Comment by mwmentor** [comment]
I see no issues.  Thanks.

**Review by mwmentor — APPROVED** [review]
All good, thanks.

File: docs/refarch/commonTermsConventions.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+| UC | Use Case: A Use Case is a list of actions or steps describing interactions between an Actor (human or external system) and a system to achieve a particular objective. An example of a Use Case from Mojaloop is "Perform Transfer with Payee Confirmation".<br/><br/>(*Further reading: "Use Case" (Definition) - From Wikipedia, the free encyclopedia*)|
+| Sync                                                              | Synchronous communications, one or two way, part of the originating process. Signified by a solid line in the UC schematics. Typically used for Messages that need to be included in a particular UC workflow in order for it to execute successfully. An example of synch Messaging is a query from the Transfers BC to the Participant Lifecycle Management BC to get Participant data if not cached to complete a Transfer request.|
```
Comment (mdebarros) [inline docs/refarch/commonTermsConventions.md:13]:
```suggestion
| Sync                                                              | Synchronous communications, one or two way, part of the originating process. Signified by a solid line in the UC schematics. Typically used for Messages that need to be included in a particular UC workflow in order for it to execute successfully. An example of sync Messaging is a query from the Transfers BC to the Participant Lifecycle Management BC to get Participant data if not cached to complete a Transfer request.|
```

File: docs/refarch/commonTermsConventions.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+| Async                                                             | Asynchronous communications, one or two way, not part of the originating process. Signified by a dotted line in the UC schematics. Typically used for Events which are used to indicate something that has happened, and is immutable and won't change, for example callback reports.|
+| POST                                                              | Utilized to create new resources. In particular, it's used to create subordinate resources. That is, subordinate to some other (e.g. parent) resource. IOW, when creating a new resource, POST to the parent and the service takes care of associating the new resource with the parent, assigning an ID (new resource URI), etc. On successful creation, the system will return a Location header with a link to the newly-created resource with the 201 HTTP status.<br/><br/>(*Source: Restful API Tutor - see References and further reading*)|
```
Comment (mdebarros) [inline docs/refarch/commonTermsConventions.md:15]:
```suggestion
| POST                                                              | Utilized to create new resources. In particular, it's used to create subordinate resources. That is subordinate to some other (e.g. parent) resource. IOW, when creating a new resource, POST to the parent and the service takes care of associating the new resource with the parent, assigning an ID (new resource URI), etc. On successful creation, the system will return a Location header with a link to the newly-created resource with the 201 HTTP status.<br/><br/>(*Source: Restful API Tutor - see References and further reading*)|
```

Two things here:

1. `IOW` <-- please define
2. `(*Source: Restful API Tutor - see References and further reading*)` <-- this applies to the entire table, but why are we not using the `[^1]` plugin to handle these?

**Review by mdebarros — CHANGES_REQUESTED** [review]
Minor comments for your attention

File: docs/refarch/commonTermsConventions.md (hunk 2 above)
Comment (mwmentor) [reply docs/refarch/commonTermsConventions.md:15]: 1. IOW = In other words   -  I can update and repost.  Please advise if that is required, thanks.
2. The entire document needs a Further Reading list and/or bibliography.  I would like to do this in much the same way as I have for the Common Terms and Conventions Used.  I need to chat to you about this.  We can chat either today or tomorrow.

Thanks. -M.

**Comment by pedrosousabarreto** [comment]
This change is already in a previous PR. Closing.

**Pull request closed without merging**